app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,188 @@
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+@font-face {
+  font-family: 'Lobster';
+  src: url('Lobster-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'Proxima Nova';
+  src: url('Proxima-Nova.otf');
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.page-container {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #f0f3f2;
+}
+
+a {
+  text-decoration: none;
+}
+
+#splash-title {
+  font-family: 'Lobster', sans-serif;
+}
+
+.splash-button {
+  font-family: 'Proxima-Nova', sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#login {
+  margin-top: 150px;
+  background-color: #3978c3;
+  width: 90vw;
+  height: 6%;
+  border-radius: 3px;
+  color: #c7def1;
+}
+
+#signup {
+  margin-top: 20px;
+  color: #9d9fa2;
+}
+
+.login-page {
+  font-family: 'Proxima-Nova', sans-serif;
+  height: 100vh;
+  width: 100vw;
+}
+
+.login-top {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #336bac;
+  height: 10%;
+  padding-left: 10px;
+  padding-right: 10px;
+  color: #c6d6ed;
+  font-weight: 300;
+}
+
+.login-title {
+  font-size: 20px;
+  font-weight: 300;
+}
+
+.login-links {
+  color: #c6d6ed;
+}
+
+#back-button {
+  font-size: 20px;
+}
+
+.new_user {
+  height: 100vh;
+  margin-top: 3%;
+  background-color: #f6f5f8;
+}
+
+.field {
+  height: 13vw;
+}
+
+input {
+  width: 100vw;
+  height: 100%;
+  border: 1px solid #f2f1f3;
+  background: #fff;
+}
+
+input::placeholder {
+  font-size: 15px;
+  padding: 4px;
+  color: #d0d2d4;
+}
+
+.actions {
+  color: #c6c9ca;
+  font-size: 20px;
+  text-align: center;
+  margin-top: 3%;
+}
+
+#submit-button {
+  border: none;
+  background: none;
+}
+
+#signout {
+  background: none;
+  border: none;
+}
+
+#group-index {
+  background: #f0f3f2;
+}
+
+.group-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  color: #bbb;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  margin: 3%;
+}
+
+.group-icon {
+  width: 80px;
+}
+
+.group-name {
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.add-button {
+  background-color: #5fb523;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 10%;
+  margin-top: 20%;
+}
+
```

Comment:
- [ ] Kindly follow the design of the template by aligning the add button at the bottom.

![image](https://user-images.githubusercontent.com/65068771/164564548-e96ad365-c6a7-40e8-8314-c9e5eec81068.png)

![image](https://user-images.githubusercontent.com/65068771/164564668-4fe0f31a-d172-4908-bd4c-b4b76c19ae28.png)

## [OPTIONAL]

- [ ] I would really suggest you remove the double notification when a transaction is done.

![image](https://user-images.githubusercontent.com/65068771/164564853-7acdf10f-8a41-4a71-850a-f21cdae0bae0.png)